drivers/openstack/openstack.go

```diff
@@ -175,13 +173,6 @@ func GetCreateFlags() []cli.Flag {
 			Usage: "OpenStack SSH port",
 			Value: 22,
 		},
-		// Using a StringFlag rather than a BoolFlag because
-		// the BoolFlag default value is always false
-		cli.StringFlag{
-			Name:  "openstack-docker-install",
```

Comment:
Thanks.  I agree we need a way to disable.  Perhaps a global flag now that
it's centralized?  Sorry for updating the openstack one -- I wanted to at
least be consistent across the providers as far as what was provisioned.

On Mon, Feb 23, 2015 at 5:06 PM, Guillaume Giamarchi <
notifications@github.com> wrote:

> In drivers/openstack/openstack.go
> <https://github.com/docker/machine/pull/590#discussion_r25207563>:
>
> > @@ -175,13 +173,6 @@ func GetCreateFlags() []cli.Flag {
> >  			Usage: "OpenStack SSH port",
> >  			Value: 22,
> >  		},
> > -		// Using a StringFlag rather than a BoolFlag because
> > -		// the BoolFlag default value is always false
> > -		cli.StringFlag{
> > -			Name:  "openstack-docker-install",
>
> The README should be updated to remove this option.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/590/files#r25207563>.
>